server/tests/landing_page_test.py

```diff
@@ -69,6 +73,20 @@ def test_chart_config_transform(self):
                 expected = json.load(f)
                 assert expected == result
 
+    def test_menu_text(self):
+        chart_config = libutil.get_chart_config()
+        spec = landing_page.build_spec(chart_config, False)
```

Comment:
i worry that this test will pass, even though the page, in english, will have different strings (e.g. if we are in the middle of updating titles). i was able to test the pybabel flow e2e in static_test.py, but that required loading a full request through the app. there is a ton of mocking to do though, in that case.

maybe if the test was renamed to test_menu_hierarchy?

Reply:
Updated the file. I don't fully understand the context here, but it would be nice to test this in english and another language setting.

Reply:
since this bypases fill_translation in landing_page.py, it skips pybabel altogether and will read the title directly from the config, not the string files (that's what gettext does). in prod, strings are read from the .mo file even for english.

+1 to testing this for a couple of languages